dpxdt/client/capture.js

```diff
@@ -77,6 +76,12 @@ if (config.cookies) {
     });
 }
 
+// Add username and password as a parameter for HTTP basic auth
+if (config.httpUserName && config.httpPassword) {
```

Comment:
Thanks for your reply. Indeed, I will  do it.



On Wed, Feb 11, 2015 at 9:29 AM, Brett Slatkin <notifications@github.com>
wrote:

> In dpxdt/client/capture.js
> <https://github.com/bslatkin/dpxdt/pull/129#discussion_r24479449>:
>
> > @@ -77,6 +76,12 @@ if (config.cookies) {
> >      });
> >  }
> >
> > +// Add username and password as a parameter for HTTP basic auth
> > +if (config.httpUserName && config.httpPassword) {
>
> Would you mind adding documentation of these parameters to README.md?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bslatkin/dpxdt/pull/129/files#r24479449>.
>